testdata/multiline-value.yaml

```
@@ -0,0 +1,4 @@
+customs:
+    - custom:
+        name: custom
+        multline: '"FOO\nBAR"'
```

I would like if this were translated into:
```suggestion
        multline: -|
            FOO
            BAR
```